cfme/tests/cloud/test_provisioning.py

```diff
@@ -33,7 +35,12 @@
 ]
 
 
-@pytest.yield_fixture(scope="function")
+@pytest.fixture()
+def vm_name():
+    return random_vm_name(context='prov')
+
+
+@pytest.fixture()
```

Comment:
this should be left as yield_fixture

Reply:
for `pytest` > version 3, `yield_fixture` and just `fixture` would work as just same, just doing `yield` would make fixture yield fixture. 
@lkhomenk any reasons why it should be left like this?

Reply:
no. you are right. it's deprecated. the only reason to keep it might be some readability, but i'm not sure it's needed here